scraper.rb

```diff
@@ -3,16 +3,21 @@
 
 agent = Mechanize.new
 
-def scrape_page(page)
+def scrape_page(page, base_url)
   table = page.at("table.ContentPanel")
-  
-  table.search("tr")[1..-1].each do |tr|
+  tr_elems = table.search("tr")
+
+  tr_elems.each_with_index do |tr, index|
+    next if index == 0 #Skipping the first beacause it's got <th> not <td>
+
+    this_application_details = tr.css('td a')[0].attributes['href']
```

Comment:
Clearer to use `.first` instead of `[0]` here https://github.com/bbatsov/ruby-style-guide#first-and-last